docs/book/src/reference/versions.md

```diff
@@ -52,7 +52,10 @@ These diagrams show the relationships between components in a Cluster API releas
 | Kubernetes v1.20 | ✓                                | ✓                             | ✓                                | ✓                             | ✓                                | ✓                             |
 | Kubernetes v1.21 | ✓                                | ✓                             | ✓                                | ✓                             | ✓                                | ✓                             |
 | Kubernetes v1.22 |                                  | ✓                             | ✓                                | ✓                             | ✓                                | ✓                             |
-| Kubernetes v1.23 |                                  |                               | ✓                                | ✓                             | ✓                                | ✓                             |
+| Kubernetes v1.23* |                                  |                               | ✓                                | ✓                             | ✓                                | ✓                             |
+
+\* There is an issue with CRDs in Kubernetes v1.23.{0-2}. ClusterClass with patches is affected by that (for more details please see [this issue](https://github.com/kubernetes-sigs/cluster-api/issues/5990)). Therefore we recommend to use Kubernetes v1.23.3+ with ClusterClass.
```

Comment:
Also, should the header of the table have CAPI v1beta1 (v1.0)  --> CAPI v1beta1 (v1.{0,1}) (or v1.x)
Not strictly related to this PR, but useful for the release...